docs/cmd/kn_service_delete.md

```diff
@@ -24,8 +24,11 @@ kn service delete NAME [flags]
 ### Options
 
 ```
+      --async              DEPRECATED: please use --no-wait instead. Delete service and don't wait for it to become ready.
```

Comment:
don't add that, not deeded to add a deprecated option.

Reply:
It's still part of `AddConditionWaitFlags()` and I wish I could create overloaded method...:) But of course, I agress it shouldn't be created with deprecated flag.

Reply:
yep, I've seen. its ok.